pkg/minishift/registration/redhat.go

```diff
@@ -51,12 +62,28 @@ func (registrator *RedHatRegistrator) CompatibleWithDistribution(osReleaseInfo *
 	}
 }
 
+func (registrator *RedHatRegistrator) SSHCommand(cmd string) (string, error) {
+	startTime := time.Now()
+	out, err := registrator.SSHCommander.SSHCommand(cmd)
+	elapsed := time.Since(startTime)
+
+	if glog.V(2) {
+		if spinnerView != nil {
```

Comment:
@hferentschik , here is issue. 

During registration process, we show spinning wheel and if we don't stop it it will appear as below in debug logs: 
```
/ Command blah blah tok 12.4 s
```

Now, for other ssh commands like `sudo -E subscription-manager version` we are not using spinning wheel, hence I need to check if reference is not nil then only stop the wheel.

Two options I can think of:
- Keep the code as it is as it is necessary to remove the spinning wheel char `/` to remove/truncate
- We don't care showing spinning wheel char `/` in debug logs.